resources/views/celebrities/show.blade.php

```diff
@@ -0,0 +1,35 @@
+<x-app-layout>
+    <x-slot name="header">
+        Celebrity Profile
+    </x-slot>
+        <div class="flex grid">
+            <div>
+                Name: {{$celebrity->name}}
+            </div>
+
+            <div>
+                <div>
+                    D.O.B: {{$celebrity->dob}}
+                </div>
+                <div>
+                    Nationality: {{$celebrity->nationality}}
+                </div>
+            </div>
+
+            <div>
+                <div>
+                    Bio: {{$celebrity->bio}}
```

Comment:
Missing a closing div here. Make sure you format your code regularly to avoid issues and bugs ;D 

The VScode shortcut to format a file is:
` Format Document (Ctrl+Shift+I)`